manifests/kots-config.yaml

```diff
@@ -3,297 +3,703 @@ apiVersion: kots.io/v1beta1
-          hidden: false
-          readonly: false
+      - name: loadBalancerSupportedKind
+        title: Load Balancer kind
+        type: textarea
+        hidden: true
+        default: |-
+          repl{{ $k8sDistribution := Distribution }}
+          repl{{ $result := fromJson "{}" }}
+          repl{{ if eq $k8sDistribution "gke" }}
+          repl{{    $_ := set $result "name" "routerLoadBalancerGKE" }}
+          repl{{    $_ := set $result "title" "GCP GKE LoadBalancer" }}
+          repl{{ else if eq $k8sDistribution "eks" }}
+          repl{{    $_ := set $result "name" "routerLoadBalancerEKS" }}
+          repl{{    $_ := set $result "title" "AWS EKS LoadBalancer" }}
+          repl{{ else if eq $k8sDistribution "aks" }}
+          repl{{    $_ := set $result "name" "routerLoadBalancerAKS" }}
+          repl{{    $_ := set $result "title" "Azure AKS LoadBalancer" }}
+          repl{{ else }}
+          repl{{    $_ := set $result "name" "unsupported" }}
+          repl{{    $_ := set $result "title" (print "ERROR: No Load Balancer supported for your kubernetes platform, " $k8sDistribution) }}
+          repl{{ end }}
+          repl{{ toJson $result }}
+      - name: errorMessage
+        type: heading
+        when: '{{repl and (ConfigOptionEquals "accessToSHModeNotKurl" "accessToSHMode_LoadBalancer") (eq (fromJson (ConfigOption "loadBalancerSupportedKind") | dig "name" "unsupported") "unsupported") }}'
+        title: '{{repl fromJson (ConfigOption "loadBalancerSupportedKind") | dig "title" "Unsupported" }}'
+      - name: accessToSHDomain
+        title: Domain
+        help_text: |-
+          The domain used to access your Self-Hosted (without protocol)
+          This is the domain that the final users will use to access your Self-Hosted.
+        type: text
+        required: true
+        default: "carto3-onprem.lan"
+      # k8s
+      - name: accessToSHModeNotKurl
+        title: Self-Hosted access mode
+        help_text: |-
+          How do you want to access to your Self-Hosted?
+        type: select_one
+        when: '{{repl not (IsKurl) }}'
+        required: true
+        default: accessToCartoDefaultLB
+        items:
+          - name: accessToCartoDefaultLB
+            title: |-
+              Default Access
+          - name: accessToCartoCustom
+            title: |-
+              Custom
+      - name: accessToCartoDefaultLBLabel
+        type: label
+        when: '{{repl (ConfigOptionEquals "accessToSHModeNotKurl" "accessToCartoDefaultLB") }}'
+        title: |-
+          ℹ️ We will deploy an external Load Balancer to access to your Carto Self-Hosted environment.
+      - name: accessToCartoCustomLabel
+        type: label
+        when: '{{repl (ConfigOptionEquals "accessToSHModeNotKurl" "accessToCartoCustom") }}'
+        title: |-
+          ℹ️ You should create and configure your own Load Balancer with a valid certificate and connect it to the CARTO router service.
+      # TLS
```

Comment:
We should finish that section during that sprint, right?

Reply:
yes, it is pending the managed certificates and the ingress config